lib/OrdinaryDiffEqRosenbrock/src/rosenbrock_caches.jl

```diff
@@ -1,13 +1,49 @@
 abstract type RosenbrockMutableCache <: OrdinaryDiffEqMutableCache end
-abstract type RosenbrockConstantCache <: OrdinaryDiffEqConstantCache end
-
-# Fake values since non-FSAL
```

Comment:
why was this deleted?

Reply:
Actually I wanted to club the constant caches into a single RosenbrockConstantCache and the names were conflicting. So I decided to keep one.